README.md

```diff
@@ -498,7 +498,7 @@ generated by the [ScalaPB compiler](https://github.com/scalapb/ScalaPB).
       <td><code>deps</code></td>
       <td>
         <p><code>List of labels, required</code></p>
-        <p>List of proto dependencies that this target depends on. Must be of type <code>proto_library</code></p>
+        <p>List of dependencies for this target. All targets of type <code>proto_library</code> are passed to <code>scalapb</code> for compilation. All other targets are used to compile the final output scala library</p>
```

Comment:
can these really be any generic target if not proto?

Is it anything that you might need to do the `with_java`, is that it?

Reply:
Well the reason I changed this is for `with_java` but you could potentially use this to pass other dependencies to the output scala_library (though not sure why one would want that).